app/src/main/java/com/example/themoviesdb/TheMoviesDBApp.kt

```diff
@@ -0,0 +1,29 @@
+package com.example.themoviesdb
+
+import android.app.Application
+import dagger.hilt.android.HiltAndroidApp
+import retrofit2.Retrofit
+import retrofit2.Retrofit.Builder
+import retrofit2.converter.gson.GsonConverterFactory
+
+@HiltAndroidApp
+class TheMoviesDBApp : Application() {
+
+    companion object {
+        private lateinit var moviesDBRetrofit: Retrofit
```

Comment:
you probably need this variable to make a singleton instance of Retrofit, but since you're using Hilt, you can use @Singletin annotation on Retrofit provider method instead.